### PR TITLE
Update of Swedish FAQ

### PR DESCRIPTION
**Pull request by matsduf**
This is an updated translation of the FAQ. It resolves issue #90.

### CODE REVIEW

File: docs/FAQ/gui-faq-sv.md [hunk 1]
```diff
@@ -94,28 +97,27 @@ Om vi utgår från att domänen du försöker testa faktiskt existerar så finns
...
+2. Eftersom Zonemaster är designad för att testa domäner (som zonemaster.net) och inte värdnamn i en domän
+(som www.zonemaster.net) så kommer Zonemaster att rapportera det som ett misslyckande om man försöker
+att testa ett domännamn som inte motsvarar en DNS-zon.
 
 #### 11. Vilken typ av DNS-frågor genererar Zonemaster? <a name="q11"></a>
 
-Det här är en svår fråga att svara på eftersom Zonemaster kommer att generera olika typer av anrop beroende på hur dina DNS-servrar svarar. Det enklaste sättet att se exakt vad Zonemaster testar är att köra ”zonemaster-cli” CLI-kommandot. Resultatet ger grundlig information om vad som händer under testet. Det bör dock nämnas att utmatningen från CLI-verktyget är väldigt tekniskt utmanande så ifall du inte gillar bits och bytes kanske du vill undvika det.
+Det här är en svår fråga att svara på eftersom Zonemaster kommer att generera olika typer av anrop
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-sv.md]: I think the Swedish reply is better than the English. In this case I just updated the current Swedish reply.

File: docs/FAQ/gui-faq-sv.md [hunk 2]
```diff
@@ -94,28 +97,27 @@ Om vi utgår från att domänen du försöker testa faktiskt existerar så finns
...
+beroende på hur dina DNS-servrar svarar.
 
-#### 12. Vad är ett odelegerat domäntest? <a name="undelegated"></a>
+Det enklaste sättet att se exakt vad Zonemaster testar är att köra ”zonemaster-cli”, CLI-kommandot
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-sv.md]: Same, same. Updated.

File: docs/FAQ/gui-faq-sv.md [hunk 3]
```diff
@@ -94,28 +97,27 @@ Om vi utgår från att domänen du försöker testa faktiskt existerar så finns
...
+(och för att göra det så måste du ladda ner och installera det) och välja full utskrift.
+Resultatet ger grundlig information om vad som händer under testet, men är också tekniskt utmanande.
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-sv.md]: Yes.

File: docs/FAQ/gui-faq-sv.md [hunk 4]
```diff
@@ -94,28 +97,27 @@ Om vi utgår från att domänen du försöker testa faktiskt existerar så finns
...
+#### 12. Vad är ett odelegerat domäntest? <a name="q12"></a>
 
-#### 13. Hur kan jag testa en domän som är en baklängesuppslagningsdomän? <a name="q13"></a>
+Ett odelegerat domäntest är ett test som genomförs på en domän som kan (men inte måste) vara fullständigt publicerad i DNS. Detta kan vara mycket användbart om du tänker flytta din domän från en registrar till en annan. Låt oss ta som exempel att din domän example.se ska flyttas från namnservern ’ns.nic.se’ till namnservern ’ns.iis.se’. I detta fall skulle du kunna köra ett odelegerat domäntest på domänen (example.se) med den namnservern du ska flytta till (ns.iis.se) innan du genomför själva flytten. När testet visar grönt så kan du vara tämligen säker på att den nya hemvisten för din domän åtminstone vet att den ska svara på frågor om din domän. Det kan emellertid fortfarande finnas fel i zoninformationen som detta test inte känner till.
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-sv.md]: 1. Ok. Done.

2. Ok. Done.

3. Ok.

4. Updated.

File: docs/FAQ/gui-faq-sv.md [hunk 5]
```diff
@@ -12,43 +12,47 @@ Zonemaster
...
 #### 1. Vad är Zonemaster? <a name="q1"></a>
 
-Zonemaster är ett program designat för att hjälpa människor att kontrollera, mäta och förhoppningsvis också bättre förstå hur DNS, domain name system, fungerar. Zonemaster består av 3 huvuddelar: 
-1. Motorn (all kod som genomför alla tester), 
-2. Kommandoradsinterfacet (CLI) samt,
-3. En server som låter dig köra zonmastertester och spara resultat med ett JSON-RPC API,
-4. Webbgränssnittet. 
+Zonemaster är ett program som är framtaget för att hjälpa människor att kontrollera, mäta och förhoppningsvis också bättre förstå hur DNS, domain name system, fungerar. Zonemaster består av 3 huvuddelar: 
+1. "Engine" - en testmotor som genomför alla DNS-tester. 
+2. Kommandoradsgränssnitt (CLI) till testmotorn.
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-sv.md]: Agree. Fixed.

File: docs/FAQ/gui-faq-sv.md [hunk 6]
```diff
@@ -94,28 +97,27 @@ Om vi utgår från att domänen du försöker testa faktiskt existerar så finns
 
 1. För att förhindra att flera test görs samtidigt på samma zon från samma IP-adress finns det en påtvingad fördröjning på 5 minuter mellan identiska test. Detta innebär att du inte kan testa en domän oftare än var 5:e minut. Om du testar din domän igen innan 5 minuter förflutit så visas det senast sparade resultatet.
 
-2. Eftersom Zonemaster är designad för att testa domäner (som zonemaster.net) och inte värdnamn i en domän (som www.zonemaster.net) kontrollerar Zonemaster webbsida domänen du skrivit in innan den skickas vidare till Zonemasters testmotor för att se att det verkligen är en domän. Denna kontroll kan i vissa sällsynta fall misslyckas (och zonen således inte godkänns som korrekt). De enda gånger vi sett detta hända är ifall de DNS-servrar som tillhör den zon du försöker testa är väldigt trasiga. Hör gärna av dig ifall detta hänt dig så vi får mer information om hur vi kan korrigera hur detta test av domänen utförs. Det här testet kommer att förbättras, det lovar vi.
+2. Eftersom Zonemaster är designad för att testa domäner (som zonemaster.net) och inte värdnamn i en domän
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply docs/FAQ/gui-faq-sv.md]: I really like the new wording.

**Pull request merged**